service/src/main/java/bio/terra/workspace/service/workspace/flight/DeleteProjectStep.java

```diff
@@ -26,12 +26,12 @@
  */
 public class DeleteProjectStep implements Step {
```

Comment:
nit: maybe rename this step to `DeleteGcpProjectStep`. We may need to come up with some step naming rules.